Tests/Resources/App/config/config.yml

```diff
@@ -1,2 +1,5 @@
 phpcr_migrations:
     version_node_name: "jcr:migrations"
+
+framework:
+    annotations: true
```

Comment:
This change was required because the CI did fail without it: https://github.com/dantleech/phpcr-migrations-bundle/runs/6258531587?check_suite_focus=true

Not sure if anything could be handled better here in doctrine phpcr to avoid this kind of error or to throw a better error message.